noisy-simulator/src/state_vector_simulator/tests.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use super::StateVectorSimulator;
+use crate::tests;
```

Comment:
It seems like all the tests so far for both noisy simulators are done using noiseless operators. It would be good to have a few tests that use operators with a simple noise model (e.g. X gate with a 0.02 probability of not affecting the state or something similar).

Reply:
I added a noisy test with a noisy identity test, just to check the noisy nature of the simulator works. Not sure if it is valuable adding more noisy tests for three reasons:
 1. The noisy behavior is already tested with that one test and the math behind the simulator is already tested in the noisy tests.
 2. The noisy tests are really slow to run. The one I added takes ~5s to run on each simulator (~10s in total).